### DISCUSSION TITLE
Inline Autofill with FlorisBoard

### DISCUSSION THREAD

**Discussion by E0697** [Kunzisoft/KeePassDX#1352] [Q&A]
Anyone using FlorisBoard/AnySoftKeyboard with inline autofill enabled?
Is it working properly?

**Comment by IzumiSenaSora** [comment]
Don't Know About AnySoftKeyboard
But for FlorisBoard ... Yes ... working fine... using it for a while.... 
Here's a preview
![Screenshot_20220610-202337_Bromite](https://user-images.githubusercontent.com/61591769/173086638-207066e7-bce4-42ca-94c9-f418d8e27bbe.png)

**Reply by E0697** [reply]
I'm currently using FlorisBoard, it shows the prompt like in the screenshot, but after unlocking it doesn't add the entries to the bar.
Is it the same for you? And what's the version of FlorisBoard you're using?
Thanks.

**Reply by IzumiSenaSora** [reply]
Nope... its working fine for me... 
I am using 0.3.15

**Reply by E0697** [reply]
Good for you.
One last question, are you using Custom/Stock Android, and which one?
I suspect this might has something to do with it.

**Reply by IzumiSenaSora** [reply]
LineageOS 18.1
